bin/anthology/papers.py

```diff
@@ -67,6 +67,8 @@ def from_xml(xml_element, *args):
         if paper.is_volume or is_journal(paper.full_id):
             del paper.attrib['xml_booktitle']
 
+        paper.attrib['booktitle'] = paper.get('booktitle')
```

Comment:
That looks like a no-op to me?

(`paper.get('booktitle')` returns `paper.attrib['booktitle']`)

Reply:
Hmm, you're right. This field wasn't getting dumped to the YAML, but I think the fix below to `get_booktitle()` must have fixed that.